src/main/drivers/max7456_symbols.h

```diff
@@ -34,49 +29,49 @@
 // Degrees Icon for HEADING/DIRECTION HOME
 #define SYM_DEGREES 0xBD
 
-// Direction arrows
-#define SYM_ARROW_UP    0x60
+// Direction arrows 
+#define SYM_ARROW_SOUTH 0x60
```

Comment:
I just copied the original MWOSD symbols file. More importantly the all of the actual symbols have left/right switch compared to the betaflight character map. My test rig has no compass, so it's difficult to check whether it's still working after my changes.